ucm2/HdmiLpeAudio/Hdmi.conf

```diff
@@ -0,0 +1,79 @@
+# Use case Configuration for sof-hda-dsp
+
+If.hdmi1 {
+	Condition { Type String "Intel HDMI/DP LPE Audio" }
```

Comment:
This is not a correct syntax and it cannot work. Please, keep the original condition `{ Type String Empty "" }`.

Also, please, post URL for you `alsa-info.sh --no-upload` context so we can add this system to the verification. Thank you.

Reply:
Here is the alsa-info https://gist.github.com/starnight/370aab1d670eaf227785e594482e203b

Reply:
Thanks to @perexg's notification!
The commit ("ucm2/HdmiLpeAudio: Enable the HDMI audio of ECS EF20") should be dropped.  Because, I found the HDMI audio part works fine with original UCM configuration.  Maybe, the HDMI cable is not connected well when I tested before.

Reply:
I dropped the commit ("ucm2/HdmiLpeAudio: Enable the HDMI audio of ECS EF20").

Reply:
I confirmed the commit ("chtrt5645: Enable Internal MIC of ECS EF20EA") is needed to enable the internal MIC.